jerry-core/include/jerryscript-core.h

```diff
@@ -522,6 +522,8 @@ jerry_length_t jerry_get_typedarray_length (jerry_value_t value);
 jerry_value_t jerry_get_typedarray_buffer (jerry_value_t value,
                                            jerry_length_t *byte_offset,
                                            jerry_length_t *byte_length);
+jerry_value_t jerry_json_parse (const char * json_string);
+jerry_value_t jerry_json_stringfy (const jerry_value_t object_to_stringify);
```

Comment:
`JSON.parse` and `JSON.stringify` take a couple of optional arguments.
I guess they could be exposed as different C APIs (i.e. `jerry_json_parse_with_reviver`, etc)

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify